build.sbt

```diff
@@ -5,53 +5,52 @@
 lazy val `routeguide-protocol` = project
   .in(file("routeguide/protocol"))
   .settings(libraryDependencies ++= Seq(mu("mu-rpc-monix")))
-  .settings(coverageEnabled := false)
-  .settings(noPublishSettings)
-  .settings(moduleName := "mu-rpc-example-routeguide-protocol")
+  .disablePlugins(SrcGenPlugin)
```

Comment:
🤔 should we make the plugin _Manual_? It's not related to this PR, but I guess is because it's looking for the `idlType`

Reply:
Yes I think we should